cmd/kube-scheduler/app/server_test.go

```diff
@@ -249,13 +250,18 @@ leaderElection:
 			wantPlugins: map[string]*config.Plugins{
 				"default-scheduler": func() *config.Plugins {
 					plugins := defaults.ExpandedPluginsV1.DeepCopy()
+					// With this (and only this?!) config comes DynamicResources after DefaultPreemption.
+					plugins.PreEnqueue.Enabled[1], plugins.PreEnqueue.Enabled[2] = plugins.PreEnqueue.Enabled[2], plugins.PreEnqueue.Enabled[1]
+					plugins.PostFilter.Enabled[0], plugins.PostFilter.Enabled[1] = plugins.PostFilter.Enabled[1], plugins.PostFilter.Enabled[0]
```

Comment:
Why are you modifying the plugins' order this way, but not explicitly as below?
```suggestion
					plugins.PreEnqueue.Enabled = []config.Plugin{
						...
					}
					plugins.PostFilter.Enabled = []config.Plugin{
						...
					}
```

Reply:
If you prefer to avoid swapping, then I can also do that, for one or the other config variant.

I chose swapping to highlight that only the order of the entries from `defaults.ExpandedPluginsV1` is different, but not the entries.